src/store/mutations/mutations-model.ts

```diff
@@ -69,7 +69,6 @@ export const mutateEffectiveDateTimeString = (state: StateIF, dateTime: string)
 
 export const mutateEffectiveDateValid = (state: StateIF, valid: boolean) => {
   state.stateModel.newAlteration.validFlags.isValidEffectiveDate = valid
-  if (!state.stateModel.tombstone.ignoreChanges) mutateHaveChanges(state, true)
```

Comment:
This is kind of an "internal" change, so don't set the HaveChanges flag.

(This flag will be set when the user sets the filing to immediate, or correctly enters effective date + time.)